http_server/www/logout.php

```diff
@@ -5,13 +5,33 @@
 
 header("Content-type: text/plain");
 
+function is_from_game(){
+ if (isset($_SERVER["HTTP_X_REQUESTED_WITH"]) && !empty($_SERVER["HTTP_X_REQUESTED_WITH"])){
+  if (strpos($_SERVER["HTTP_X_REQUESTED_WITH"], "ShockwaveFlash/") !== 0){
+   return true;
+  }
+     else
+  {
+      return false;   
+  }
+ }
+ else
+ {
+    return false;
+ }
+}
+
```

Comment:
The thought of using `HTTP_X_REQUESTED_WITH` crossed my mind; However, this too can be easily spoofed.